docs/kafka/consumer-configuration-kafka/README.adoc

```diff
@@ -291,11 +291,12 @@ Use the {product-kafka} web console to select consumer groups and reset partitio
 A reset changes the offset position from which consumers read from the message log of a topic partition.
 To reset an offset position, the consumer group must have _NO MEMBERS_ connected to a topic.
 
-Choose one of the following options for the reset:
+Choose one of the following options for a *New offset*:
 
-* *Earliest* resets to the earliest offset at the start of the message log.
-* *Latest* resets to the latest offset at the end of the message log.
-* *Custom offset* resets to a specific offset in the message log.
+
+* *absolute* resets to a specific offset in the message log.
```

Comment:
Are these names definitely in lowercase in the UI?

Reply:
yes I generated the steps, that's why I changed to lowercase